calabash-cucumber/lib/calabash-cucumber/launch/simulator_launcher.rb

```diff
@@ -251,13 +251,24 @@ def app_bundle_or_raise(path=nil, device_build_dir='iPhoneSimulator')
             puts('-'*37)
           end
         else
-          bo_dir = build_output_dir_for_project
-          sim_dirs = Dir.glob(File.join(bo_dir, '*-iphonesimulator', '*.app'))
+          search_dir = build_output_dir_for_project || DERIVED_DATA
+          sim_dirs = ''
+          apps = `find #{search_dir} -name "*.app" | sort -n`.split("\n")
```

Comment:
@ark-konopacki 

Interesting!  Why sort numerically (`-n`)? 

We want to search for the modification time and take the newest.

```
find #{search_dir} -type d -name "*.app" -exec stat -f "%m %N" {} \; | sort -rn | cut -d" " -f2
```